editor/components/inserter/inline-menu.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { NavigableMenu } from '@wordpress/components';
+import { BlockIcon } from '@wordpress/blocks';
+
+export default class InserterInlineMenu extends Component {
+	render() {
+		return (
+			<NavigableMenu>
+				<div
+					className="editor-inserter__separator"
+					id="editor-inserter__separator-inline"
+					aria-hidden="true"
+				>
+					Inline Blocks
```

Comment:
This text should be wrapped with `i18` helper and another one a few lines later, too.

Reply:
Thanks for catching that! Fixed in https://github.com/WordPress/gutenberg/pull/5794/commits/5275914f57857e2a6109989cb04c3159401985d9.